ci/tasks/windows/bbl-up.sh

```
@@ -1,32 +1,25 @@
 #!/bin/bash
+set -eu -o pipefail
+
 main() {
-  source $PWD/bosh-dns-release/ci/assets/utils.sh
-  local output_dir="$PWD/envs-output/"
-  local bbl_state_env_repo_dir=$PWD/envs
-  trap "commit_bbl_state_dir ${bbl_state_env_repo_dir} ${ENV_NAME} ${output_dir} 'Update bbl state dir'" EXIT
+  source "$PWD/bosh-dns-release/ci/assets/utils.sh"
+
+  local bosh_release_path=$(echo "$PWD"/bosh-candidate-release/*.tgz)
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)